gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -18,13 +18,15 @@
  */
 package org.apache.iceberg.gcp;
 
+import com.google.api.client.util.Preconditions;
```

Comment:
wrong import?

Reply:
yes, this is a wrong import and should use the one from `iceberg-guava`